tests/epyccel/test_arrays_multiple_assignments.py

```diff
@@ -140,11 +140,14 @@ def f():
 
     # Check that we got exactly 1 Pyccel error
     assert errors.has_errors()
-    assert errors.num_messages() == 1
+    assert errors.num_messages() == 2
 
     # Check that the error is correct
     error_info = [*errors.error_info_map.values()][0][0]
     assert error_info.symbol  == 'x'
+    assert error_info.message == STACK_ARRAY_NON_LITERAL_SHAPE
+    error_info = [*errors.error_info_map.values()][0][1]
+    assert error_info.symbol  == 'x'
     assert error_info.message == STACK_ARRAY_DEFINITION_IN_LOOP
```

Comment:
The comment line should be updated: "the error is correct" -> "the errors are correct"

Reply:
Fixed [130f7ca](https://github.com/pyccel/pyccel/pull/993/commits/130f7ca137368cbf5575facac9c4f1c28a22a099)